sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -26,4 +27,13 @@
   @interface ValueEntity {
     Class<? extends kalix.javasdk.valueentity.ValueEntity<?>> value();
   }
+
+  @Target(ElementType.METHOD)
+  @Retention(RetentionPolicy.RUNTIME)
+  @Documented
+  @interface Topic {
+    String value();
+    // @AliasFor("value")
```

Comment:
WIP

Reply:
I tested the alias with the following as shown [here](https://www.baeldung.com/spring-aliasfor-annotation#explicit-within) but it doesn't work. I get `value` empty if I set the `name`.

``` 
    @AliasFor("name")
    String value() default "";

    @AliasFor("value")
    String name() default "";
```

As far as I understand this only works if Spring instantiates the bean which is not our case because we use reflection. As you mention @octonato in a previous call and I can see now in the code.

Reply:
yeah, I think you are right. The alias won't work in our case. 

We can leave it out for now and eventually add our own alias annotation in the future if we find it will be useful.